integration/run_test.go

```diff
@@ -77,6 +79,28 @@ func TestRunMultipleContainers(t *testing.T) {
 		},
 	}
 
+	checkOverlayFallbackCount := func(output string, expected int) error {
```

Comment:
This seems a duplicate of https://github.com/awslabs/soci-snapshotter/blob/792a8996dc02311d9ca85c23fd18ae5563212f28/integration/metrics_test.go#L121 can you extract it to a separate func and reuse here?